app/routes/apply/eligibility/claim/file-upload.js

```diff
@@ -232,11 +225,11 @@ function getTargetDir (req) {
   const decryptedReferenceId = decrypt(req.session.referenceId)
   let targetDir
   if (req.query.document !== 'VISIT_CONFIRMATION' && req.query.document !== 'RECEIPT') {
-    targetDir = `${config.FILE_UPLOAD_LOCATION}/${decryptedReferenceId}/${req.query.document}`
+    targetDir = `${decryptedReferenceId}/${req.query.document}`
```

Comment:
is this still a dir? or is it a path?

Reply:
It's a dir but technically not in the same way as local storage.